tests/cli/test_compose_ext4-filesystem.sh

```diff
@@ -16,8 +16,21 @@ set -e
 
 CLI="${CLI:-./src/bin/composer-cli}"
 
-
 rlJournalStart
+    rlPhaseStartSetup
+        if [[ -n $SUBSCRIPTION_HOSTNAME && -n $SUBSCRIPTION_USERNAME && -n $SUBSCRIPTION_PASSWORD ]]; then
+            rlRun "mkdir /etc/yum.repos.d/disabled"
+            rlRun "mv /etc/yum.repos.d/*.repo /etc/yum.repos.d/disabled"
```

Comment:
FYI you can copy the entire directory and make the backup in a single operation.